packages/xstate-test/src/types.ts

```diff
@@ -86,7 +86,7 @@ export interface TestParam<TState, TEvent extends EventObject> {
     [key: string]: (state: TState) => void | Promise<void>;
   };
   events?: {
-    [TEventType in TEvent['type']]?: EventExecutor<TState, TEvent>;
+    [TEventType in TEvent['type']]?: EventExecutor<TState, ExtractEvent<TEvent, TEventType>>;
```

Comment:
Could we add a test that would cover this change? We can fake type-level tests like here:
https://github.com/statelyai/xstate/blob/506686dbda27a6ee26ccda36eddcd89cb68fc2ab/packages/xstate-graph/test/types.test.ts

Reply:
I've added a test -- is it adequate?